test/hotspot/jtreg/gc/TestPLABAdaptToMinTLABSize.java

```diff
@@ -68,9 +69,11 @@ private static void runTest(boolean shouldSucceed, String... extraArgs) throws E
     }
 
     public static void main(String[] args) throws Exception {
-        runTest(true, "-XX:MinTLABSize=100k");
-        // Should not succeed when explicitly specifying invalid combination.
-        runTest(false, "-XX:MinTLABSize=100k", "-XX:OldPLABSize=5k");
-        runTest(false, "-XX:MinTLABSize=100k", "-XX:YoungPLABSize=5k");
+        for (String gc : Arrays.asList("-XX:+UseG1GC", "-XX:+UseParallelGC")) {
```

Comment:
I think this could be problematic if you compile out G1 or Parallel. I would suggest that you create two separate run blocks. One for G1 and one for Parallel, and then pass in the GC to test through the `args`.